release_checklist.sh

```diff
@@ -0,0 +1,242 @@
+    # strips the newlines from the body. The call below does not.
+    issue_body=$(gh issue view "$issue_number" --json 'body' --jq '.body')
+
+    issue_comment="Issue updated:"
+
+    if [[ -n "$include_aztec_steps" ]]; then
+      aztec_checklist_template=$(<"$script_path/templates/release_checklist_update_aztec.md")
+      aztec_checklist_template=${aztec_checklist_template//$'\n'/\\n}
+      issue_body=$(sed -e "s/.*optional_aztec_release.*/$aztec_checklist_template/" <<< "$issue_body")
+      issue_comment="$issue_comment"$'\n\n- Added Aztec Release Checklist'
+      confirm_message="$confirm_message"$'\n\n- Add Aztec Release Checklist'
+    fi
+
+    if [[ -n "$include_incoming_changes" ]]; then
+      incoming_change_note=$checklist_message
+      if [[ -n "$incoming_change_note" ]]; then
+        incoming_change_note="**Update:** $checklist_message"
+      fi
+      incoming_changes_template=$(sed -e "s/{{incoming_change_note}}/${incoming_change_note}/g" "$script_path/templates/release_checklist_update_incoming.md")
+      incoming_changes_template=${incoming_changes_template//$'\n'/\\n}
+
+      issue_body=$(sed -e "s/.*optional_incoming_changes.*/${incoming_changes_template}/" <<< "$issue_body")
+      issue_comment="$issue_comment"$'\n\n- Added Incoming Changes Checklist'
+      confirm_message="$confirm_message"$'\n\n- Add Incoming Changes Checklist'
+    fi
+
+    if [[ -n "$checklist_message" ]]; then
+      issue_comment="$issue_comment"$'\n\n'"$checklist_message"
+    fi
+
+    if [[ -n "$debug_template" ]]; then
+      echo "$issue_body"
+      exit 0
+    fi
+
+    if [[ -z "$auto_confirm" ]]; then
+      echo ""
+      confirm_to_proceed "$confirm_message"$'\n\n'
+    fi
+
+    gh issue comment "$issue_number" --body "$issue_comment">/dev/null
+    gh issue edit "$issue_number" --body "$issue_body"
+  popd >/dev/null
+
+  exit 0;
+fi
+
+
+
+if [[ -z "$release_type" ]]; then
+  default_release_type="scheduled"
+
+  read -r -p "Please enter release type: scheduled|beta|hotfix [$default_release_type]:" release_type
+  release_type=${release_type:-$default_release_type}
+  echo ""
+fi
+
+if [[ ! "$release_type" == "scheduled" && ! "$release_type" == "beta" && ! "$release_type" == "hotfix" ]]; then
+    abort "Error release type must be scheduled|beta|hotfix, you entered '$release_type'"
+fi
+
+if [[ -z "$version_number" ]]; then
+  pushd "$gb_mobile_path" >/dev/null
+
+  # Ask for new version number
+  current_version_number=$(jq '.version' package.json --raw-output)
+  #version_array=($(awk -F. '{$1=$1} 1' <<< "${current_version_number}"))
+  IFS='.' read -ar version_array <<< "$current_version_number"
+  default_version_number="${version_array[0]}.$((version_array[1] + 1)).${version_array[2]}"
+
+  read -r -p "Enter the new version number [$default_version_number]: " version_number
+  echo ""
+  version_number=${version_number:-$default_version_number}
```

Comment:
Similar to previous comments, I'm wondering if we could print the version number selected.

```suggestion
  version_number=${version_number:-$default_version_number}
  echo "Using '$version_number' as new version number"
  echo ""
```